### PR TITLE
feat: add organization proto

### PR DESCRIPTION
**Pull request by masaaania**
## This PR

- adds functions and messages for the organization to proto.
- implementation will be pushed in the next PR.

### CODE REVIEW

File: proto/environment/command.proto [hunk 1]
```diff
@@ -63,3 +63,31 @@ message EnableProjectCommand {}
...
 
 message ConvertTrialProjectCommand {}
+
+message CreateOrganizationCommand {
+  string name = 1;
+  string url_code = 2;
+  string description = 3;  // optional
+}
+
+message CreateTrialOrganizationCommand {
```
Comment (cre8ivejp) [inline proto/environment/command.proto:73]: What do you think about adding a flag in the `CreateOrganizationCommand` to create a trial project to reduce the APIs?

Comment (masaaania) [reply proto/environment/command.proto:73]: Nice idea 😄.

**Comment by masaaania** [comment]
@cre8ivejp 

I updated and fixed the PR. Please take a look when you have a time.

File: proto/environment/command.proto [hunk 2]
```diff
@@ -63,3 +63,28 @@ message EnableProjectCommand {}
 message DisableProjectCommand {}
 
 message ConvertTrialProjectCommand {}
+
+message CreateOrganizationCommand {
+  string name = 1;
+  string url_code = 2;
+  string description = 3;  // optional
+  string is_trial = 4;
```
Comment (cre8ivejp) [inline proto/environment/command.proto:71]: Please use boolean instead.

**Review by cre8ivejp — APPROVED** [review]
Thank you!!

File: proto/environment/command.proto (hunk 2 above)
Comment (masaaania) [reply proto/environment/command.proto]: @cre8ivejp 
Thank you, nice catch!! 🙏 🙏

**Pull request merged**